VideoStabilization/video_stabilization.py

```diff
@@ -165,4 +169,4 @@ def fixBorder(frame):
 cap.release()
 out.release()
 # Close windows
-cv2.destroyAllWindows()
\ No newline at end of file
+cv2.destroyAllWindows()
```

Comment:
Looks like you made a whitespace change in these lines. You can remove that, and then this is ready to merge.

Reply:
Whoops, I interpreted this in the opposite of how I should, resolving.